sirhelpers.h

```diff
@@ -64,32 +64,46 @@ uint16_t _sir_geterrcode(uint32_t err) {
 
 /** Even more evil macros used for binary searching arrays. */
 # define _SIR_DECLARE_BIN_SEARCH(low, high) \
-    size_t _low  = low; \
-    size_t _high = high; \
+    size_t _low  = low;                     \
+    size_t _high = high;                    \
     size_t _mid  = (_low + _high) / 2;
 
 # define _SIR_BEGIN_BIN_SEARCH() do {
-# define _SIR_ITERATE_BIN_SEARCH(comparison) \
-    if (0 == comparison) { \
-        break; \
-    } \
-    \
-    if (_low == _high) \
-        break; \
-    \
+# define _SIR_ITERATE_BIN_SEARCH(comparison)    \
```

Comment:
I'm ambivalent about this. I almost like the right-align better. Which is your preference?

Reply:
You said left after, so let's go with left.  I like right myself (which is why some of that happened, due to my editor settings).  But I'm not going to die for either :)